### PR TITLE
Br factories fix

### PR DESCRIPTION
**Pull request by riverswb**
Just fancies up our factories.

### CODE REVIEW

File: spec/support/factories/couch.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                zipcode: "12345"},
+              { city: "New Orleans",
+                state: "LA",
+                zipcode: "70112"},
+              { city: "Colorado",
+                state: "CO",
+                zipcode: "80123"}
+              ]
+
+    this_city = cities.sample
```
Comment (tmikeschu) [inline spec/support/factories/couch.rb:15]: Clever!

File: spec/support/factories/couch.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FactoryGirl.define do
+  factory :couch do
+    cities = [{
+                city: "Chicago",
+                state: "IL",
+                zipcode: "12345"},
```
Comment (tmikeschu) [inline spec/support/factories/couch.rb:6]: Is this fo real?

File: spec/factories.rb [hunk 3]
```diff
@@ -1,23 +0,0 @@
-FactoryGirl.define do
```
Comment (tmikeschu) [inline spec/factories.rb:1]: Nice move to the directory.

**Review by tmikeschu — COMMENTED** [review]
Thanks for setting us up!

**Comment by njgheorghita** [comment]
looks way better than one massive factory file, thanks!

**Pull request merged**